Misc/NEWS.d/next/Core and Builtins/2020-12-27-18-07-43.bpo-27794.sxgfGi.rst

```diff
@@ -0,0 +1,2 @@
+Add ``name`` attribute to ``property`` class to add more information when
```

Comment:
Revise this to:  "Improve the error message for failed writes/deletes to property objects.  When possible, the attribute name is now shown."

Reply:
Changed, thanks